packages/gatsby-theme-store/src/components/Auth/Providers/ExternalProvider/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { Alert, Box, Button, Center, Flex, Spinner } from '@vtex/store-ui'
+import type { AuthProviderComponentProps } from '@vtex/store-ui'
+import { useIntl } from '@vtex/gatsby-plugin-i18n'
+import React, { useEffect, useState } from 'react'
+import type { FC } from 'react'
+import { useStartLogin } from '@vtex/gatsby-theme-store/src/sdk/auth/useStartLogin'
+import { api } from '@vtex/gatsby-theme-store/src/sdk/auth/Service/api'
+
+const oAuthErrorFallbackUrl = () =>
+  new URL(api.oauth.error, window.location.origin).href
+
+const oAuthCallbackUrl = () =>
+  new URL(api.oauth.finish, window.location.origin).href
+
+const oAuthRedirectUrl = (providerName: string) => {
```

Comment:
Sorry, I was supposed to fix that, those constants are using the window and should not be imported inside server rendering scope. I even commented that with @emersonlaurentino that I think it should be turned into a function so that it becomes more secure. I was supposed to change that but ended up using locally, will do that soon.